WordPress/Learn
# Add single Lesson Plan page - rebased

Same as #2540, but rebase to the trunk.
New commits in this PR start from [Rename to sidebar-meta-list](https://github.com/WordPress/Learn/pull/2645/commits/b3f91e3cced61f8c972262b4da5017e9552cfbf1)

## Screenshot

![image](https://github.com/WordPress/Learn/assets/18050944/026e475d-5b4e-464a-b46d-dda905f6b803)

## Testing

1. Check single lesson plan page functions
2. Check single tutorial page sidebar
3. Check single course page sidebar
4. Make sure everything works as expected
